src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Model/PowerShellRemotePackage.cs

```diff
@@ -35,7 +35,8 @@ internal static List<PowerShellRemotePackage> GetPowerShellPackageView(IEnumerab
                         var versions = await data.GetVersionsAsync();
                         var results = versions?.Select(v => v.Version).OrderByDescending(v => v).ToArray();
                         return results ?? Enumerable.Empty<NuGetVersion>();
-                    }, NuGetUIThreadHelper.JoinableTaskFactory)
+                    }, NuGetUIThreadHelper.JoinableTaskFactory),
+                    LicenseUrl = data.LicenseUrl.AbsoluteUri
```

Comment:
Verify how this works for packages without a license url, this is likely null and will throw